tests/schema/tests.py

```diff
@@ -335,6 +335,20 @@ def test_text_field_with_db_index_to_fk(self):
             editor.alter_field(AuthorTextFieldWithIndex, old_field, new_field, strict=True)
         self.assertForeignKeyExists(AuthorTextFieldWithIndex, 'text_field_id', 'schema_author')
 
+    @unittest.skipUnless(connection.vendor == 'mysql', "MySQL specific")
```

Comment:
As far as I'm aware this test should work on all databases, even if it's a regression test for an issue on MySQL. Currently it fails:
- on Oracle, PostgreSQL, and **MySQL**:
```
ValueError: Found wrong number (0) of foreign key constraints for schema_author.author_id
```
- on SQLite:
```
django.core.exceptions.FieldDoesNotExist: BookWithIndex has no field named 'writer'
```